featurebyte/models/offline_store_feature_table.py

```diff
@@ -115,6 +115,7 @@ class OfflineStoreFeatureTableModel(FeatureByteCatalogBaseDocumentModel):
     precomputed_lookup_feature_table_info: Optional[PrecomputedLookupFeatureTableInfo]
     feature_store_id: Optional[PydanticObjectId] = Field(default=None)
     deployment_ids: List[PydanticObjectId] = Field(default_factory=list)
+    aggregation_ids: List[str] = Field(default_factory=list)
```

Comment:
Should we sort the `aggregation_ids`?

Reply:
That shouldn't be needed as we won't be querying this field by exact match, so the ordering doesn't matter.